features/networking/multus.feature

```diff
@@ -490,12 +490,13 @@ Feature: Multus-CNI related scenarios
   @admin
   @destructive
   Scenario: CNO manager mavlan configured manually with static
-    # Make sure that the multus is Running
     Given the multus is enabled on the cluster
+    And I store all worker nodes to the :nodes clipboard
```

Comment:
In this case i am simply forcing test to execute on a worker node. Earlier it was on master